### ISSUE TITLE
SoundLoop interval of 0

### ISSUE THREAD

**Issue by JunShern** [processing/p5.js-sound#313]
Hi,

I've found that whenever I set

```
mySoundLoop.interval = 0 
```

for a p5.SoundLoop, the sketch freezes (presumably because it tries to run the SoundLoop infinitely quickly). However, in the case of updating the SoundLoop based on note event deltatimes, it is often useful to have an interval of 0.

The quick fix I'm using now is to just do

```
mySoundLoop.interval = max(desiredInterval, 0.01) 
```

But we might want to add this into the source code itself. Any thoughts? @therewasaguy 

Thanks!

**Comment by parulpriyedarshani** [comment]
@JunShern I agree with your solution and would like to work on this issue. After doing some research I was unable to find desiredInterval in the codebase. Could you please tell me about how this desiredInterval is working and where has it been initialized in the code?
Thanks
